doc/source/cli.md

```diff
@@ -0,0 +1,407 @@
+(cli)=
+
+# Command Line Interface (CLI)
```

Comment:
To match the British-style of the rest of the documentation, we can use lower-case "Command line interface", even for acronyms and don't need to define the acronym in the title :wink: (already defined in first sentence below)